app/task_routes.py

```diff
@@ -0,0 +1,127 @@
+from flask import Blueprint
+from app import db, SLACK_API_KEY
+from app.models.task import Task
+from flask import Blueprint, jsonify, make_response, request, abort
+from datetime import datetime
+import requests
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+# Helper Functions
+def valid_int(number, parameter_type):
```

Comment:
Great work writing this helper function. This is repeated in your goal and task routes. Consider making a separate file like `utils.py` and then importing the function where it is needed with `from app.utils import valid_int`